snapcraft/main.py

```diff
@@ -35,13 +35,15 @@
   snapcraft [options] keys
   snapcraft [options] register-key [<key-name>]
   snapcraft [options] register <snap-name> [--private]
-  snapcraft [options] sign-build <snap-file> [--key-name <key-name>] [--local]
+  snapcraft [options] sign-build <snap-file> [--key-name=<key-name>] [--local]
   snapcraft [options] upload <snap-file>
   snapcraft [options] push <snap-file> [--release <channels>]
   snapcraft [options] release <snap-name> <revision> <channel>
   snapcraft [options] list-plugins
   snapcraft [options] tour [<directory>]
   snapcraft [options] update
+  snapcraft [options] gated <snap-name>
+  snapcraft [options] validate <snap-name> [<snap-revision> ...] [--key-name=<key-name>]
```

Comment:
Better arg-name ... maybe "validation"?

Yes, it's always snap-name=revision

And it's most definitely not optional, there must be at least one. How does one tell docopt "one or more of these"?

Yes, calling "snapcraft validate telegram-sergiusens" currently does nothing. I can add a check for "at least one validation" if there's no way to tell docopt to do it.